Tests/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ macro(setup_test)
     set(PLOT_TEST ${CURRENT_TEST_BINARY_DIR}/plt00010)
     # Find fcompare
     if(PELEC_ENABLE_FCOMPARE_FOR_TESTS)
-      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/AMReX/Tools/Plotfile/fcompare)
+      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/PelePhysics/Submodules/amrex/Tools/Plotfile/fcompare)
```

Comment:
This should have the renamed `fcompare`.

Reply:
thanks, missed that one.